packages/core/test/support.ts

```diff
@@ -278,12 +278,17 @@ export function getPoolMax(): number {
   return Config[getTestDialect()].pool?.max ?? 1;
 }
 
-type ExpectationKey = 'default' | Permutations<Dialect>;
+type ExpectationKey = 'default' | Permutations<Dialect, 4>;
```

Comment:
```suggestion
// The depth of the permutation should be half of the number of supported dialects
type ExpectationKey = 'default' | Permutations<Dialect, 4>;
```
or something like that